.gitignore

```diff
@@ -1,5 +1,6 @@
 # Python compiled binaries.
 __pycache__/
+*.pyc
```

Comment:
In PY 3, all .pyc files go to `__pycache__` in the repo, or in a global cache.

So that is already covered with the rule above.

```suggestion
```

Reply:
eh it's your repo. :) Maybe because I ran from dayliopy `python clean_csv.py` rather than `python ./dayliopy/clean_csv.py`